r11701/default_common_inlists/binary/src/run_binary_extras.f

```diff
@@ -373,13 +373,13 @@ real(dp) function mass_conv_core(s)
            real(dp) :: dm_limit
            include 'formats'
            mass_conv_core = 0.0d0
-           dm_limit = s% conv_core_gap_dq_limit*s% xmstar
+           dm_limit = s% conv_core__dq_limit*s% xmstar
```

Comment:
This and the following three changes look like you replaced `gap` by nothing. I think, that was a mistake and should get reverted.